cmd/openfga/main_test.go

```diff
@@ -201,7 +210,6 @@ func TestFunctionalGRPC(t *testing.T) {
 
 	t.Run("TestCreateStore", func(t *testing.T) { GRPCCreateStoreTest(t, tester) })
 	t.Run("TestGetStore", func(t *testing.T) { GRPCGetStoreTest(t, tester) })
-	t.Run("TestListStores", GRPCListStoresTest) // run an isolated tester from the others so bootstrapped stores don't collide
```

Comment:
For some reason that I could not understand, if I leave this as is, I would get a leak reported about an HTTP or GRPC connection remaining open.